build/install_as_php_client_linux.sh

```diff
@@ -0,0 +1,175 @@
+#!/bin/bash -m
+
+set +e
+set +x
+
+SCRIPT_PATH="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+
+PROJ_FOLDER="php-client"
+
+
+apt update
+
+
+#install git, if needed
+which -s git
+if [[ $? != 0 ]] ; then
+  printf 'git was not installed.  Installing git...\n'
+  apt -y install git
+  wait
+else
+  printf 'git was already installed!\n'
+fi
```

Comment:
```suggestion
apt install -y git
```
If `git` is already installed using apt, wouldn't `apt install` just return 0?

Reply:
My idea here was to make the scripts on macOS and linux somewhat similar for people who needed to install on both mac and linux.  Also, if they have already installed say - a certain version of PHP - we don't necessarily want to force a newer version on them or get into guessing whether or not they want to upgrade it.

Keeping it in multiple steps allows easier debugging for when something fails.  Combining all into one looks smart, but it often isn't.

Having said all of that - you can alter the build scripts as u see fit.  My advice would be to just make sure you test them on both macos and linux to make sure that you don't break anything.